README.md

```diff
@@ -1 +1,42 @@
-# helloWorldProject
\ No newline at end of file
+# helloWorldProject
+In this project, I set up a "Hello world" repository. I create html, css, test files .I created a feature branche to be familiar with github , I created Pull request . .
+
+Built With
```

Comment:
### OPTIONAL 
- Kindly use this format for your headers; ` ### Getting started`,  it would make them headings, do this for all headers in your readme  👍🏿

Reply:
Thank you for your comments. 

- `I` don't know how to edit the branch name. 
- Update the Readme file